### PR TITLE
feat(backend): tables query

### PR DESCRIPTION
**Pull request by Elweyn**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/src/graphql/resolvers/TableResolver.spec.ts [hunk 1]
```diff
@@ -1448,7 +1726,7 @@ describe('TableResolver', () => {
               {
                 query,
                 variables: {
-                  tableId: -1,
+                  tableId: 25,
```
Comment (Elweyn) [inline backend/src/graphql/resolvers/TableResolver.spec.ts:1729]:
```suggestion
                  tableId: -1,
```

File: backend/src/graphql/resolvers/TableResolver.spec.ts [hunk 2]
```diff
@@ -2084,5 +2127,145 @@ describe('TableResolver', () => {
...
+              singleResult: {
+                data: {
+                  tables: [
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'My Table',
+                      public: true,
+                      users: [],
+                    },
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolvers/TableResolver.spec.ts:2192]: Why is Räuber not user in this table?
Whats the criteria for matching this table when its not users?

File: backend/src/graphql/resolvers/TableResolver.ts [hunk 3]
```diff
@@ -297,6 +297,40 @@ export class TableResolver {
     })
   }
 
+  @Authorized()
+  @Query(() => [Table])
+  async tables(@Ctx() context: Context): Promise<Table[]> {
+    const { user } = context
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolvers/TableResolver.ts:303]: maybe convert this object so you don't have to `user?.`

File: backend/src/graphql/resolvers/TableResolver.ts [hunk 4]
```diff
@@ -297,6 +297,40 @@ export class TableResolver {
...
+      include: {
+        users: true,
+      },
+    })
+    const tables: Table[] = []
+    for (const meeting of dbMeetings) {
+      const usersInMeetings = await findUsersInMeetings(meeting)
+      tables.push(new Table(meeting, usersInMeetings))
+    }
+    return tables
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolvers/TableResolver.ts:331]: pretty sure you can replace this with
`dbMeetings.map((meeting) => new Table(meeting, findUsersInMeetings(meeting))`

File: backend/src/graphql/resolvers/TableResolver.spec.ts (hunk 2 above)
Comment (Elweyn) [reply backend/src/graphql/resolvers/TableResolver.spec.ts:2192]: This is "my table" where we have a connection from user to meeting.
|------------------------|---------------------|
| User                   |  Meeting            |
| meetingId              |  userId             |
| meeting                |  user               |

For group table we have the connection through UsersInMeetings what is shown in the users.
|------------------------|---------------------|---------------------|
| User                   | UsersInMeetings     | Meeting             |
|------------------------|---------------------|---------------------|
| id                     | userId              | id                  |
|                        | meetingId           |                     |

**Review by Bettelstab — APPROVED** [review]

**Pull request merged**